views/main.html

```diff
@@ -18,17 +18,7 @@
         <script src="https://use.fontawesome.com/2155f7b139.js"></script>
     </head>
     <body id="bootstrap-override">
-        <div id="web-monitoring-ui-root">
-            <div class="row">
-                <div class="col-md-12">
-                    <div class="progress active progress-striped">
-                        <div class="progress-bar progress-success" style="width:100%" role="progressbar" aria-valuemin="0" aria-valuemax="100" aria-valuenow="100">
-                            <span>Please wait<span class="dotdotdot"></span></span>
-                        </div>
-                    </div>
-                </div>
-            </div>
-        </div>
+        <div id="web-monitoring-ui-root"></div>
```

Comment:
I didn't get this one, sorry, I removed the old loading and left it as it was because when loading the page the new loader appears